RoaringBitmap/src/main/java/org/roaringbitmap/ArrayContainer.java

```diff
@@ -375,6 +375,8 @@ public boolean equals(Object o) {
       return ArraysShim.equals(this.content, 0, cardinality, srb.content, 0, srb.cardinality);
     } else if (o instanceof RunContainer) {
       return o.equals(this);
+    } else if (o instanceof BitmapContainer) {
```

Comment:
This cannot happen. An ArrayContainer can never be equal to a BitmapContainer when the data abides by the specification.